test/uk/gov/hmrc/transitmovementsvalidator/services/JsonValidationServiceSpec.scala

```diff
@@ -234,8 +233,12 @@ class JsonValidationServiceSpec extends AnyFreeSpec with Matchers with MockitoSu
 
       whenReady(result.value) {
         e =>
-          e mustBe Left(FailedToParse("""Unexpected close marker '}': expected ']' (for root starting at [line: 1, column: 0])
-              | at [line: 73, column: 2]""".stripMargin))
+          e mustBe Left(
```

Comment:
exception should not include ```[Source: (org.apache.pekko.stream.impl.io.InputStreamAdapter);``` such part. As it will look weird to end user. Can you please check how to handle it? it should be just like the original message.